py/kubeflow/testing/notebook_tests/nb_test_util.py

```diff
@@ -0,0 +1,112 @@
+"""Some utitilies for running notebook tests."""
+
+import datetime
+import logging
+import os
+from urllib.parse import urlencode
+import uuid
+import yaml
+
+from google.cloud import storage
+from kubernetes import client as k8s_client
+from kubeflow.testing import util
+
+# This is the bucket where the batch jobs will uploaded an HTML version of the
+# notebook will be written to. The K8s job is running in a Kubeflow cluster
+# so it needs to be a bucket that the kubeflow cluster can write to.
+# This is why we don't write directly to the bucket used for prow artifacts
+NB_BUCKET = "kubeflow-ci-deployment"
+PROJECT = "kubeflow-ci-deployment"
+
+def logs_for_job(project, job_name):
+  """Get a stack driver link for the job with the specified name."""
+  logs_filter = f"""resource.type="k8s_container"
+   labels."k8s-pod/job-name" = "{job_name}"
+"""
```

Comment:
```suggestion
 """
```

Reply:
I don't think we want to do that. """ is a literal string. So if we indent line 25 we will end up including some extra whitespace in the value which we don't necessarily want.